client/signup/steps/site-type/form.jsx

```diff
@@ -51,29 +56,26 @@ class SiteTypeForm extends Component {
 							onClick={ this.handleSubmit.bind( this, siteTypeProperties.slug ) }
 						>
 							<strong className="site-type__option-label">{ siteTypeProperties.label }</strong>
-							<span className="site-type__option-description">
-								{ siteTypeProperties.description }
-							</span>
-							{ ! this.props.isJetpack && siteTypeProperties.purchaseRequired && (
-								<Badge className="site-type__option-badge" type="info">
-									{ this.props.translate( 'Purchase required' ) }
-								</Badge>
+							{ showDescriptions && (
+								<>
+									<span className="site-type__option-description">
+										{ siteTypeProperties.description }
+									</span>
+									{ siteTypeProperties.purchaseRequired && (
+										<Badge className="site-type__option-badge" type="info">
+											{ translate( 'Purchase required' ) }
```

Comment:
It seems a bit arbitrary to group this with the descriptions and show/hide with `showDescriptions`.

`isJetpack` also seems a bit surprising, but it may be better to leave that check as-is. It seems like the root of that problem is that `siteTypeProperties.purchaseRequired` is inaccurate for Jetpack sites 😕

Reply:
What do you think about `showPurchaseRequired`? I'd be in for a generic way to hide all purchase required badges because it's not really different from how hiding works with `isJetpack` check.

I also figured the badge is quite close semantically to the description text itself. :-)

Happy to add the check back if you feel like it's better that way!

Reply:
`showPurchaseRequired` seems preferable to putting them both under the same prop.

Reply:
Added `showPurchaseRequired` in b0292460434f8c6c561d41473702b181fedc9604